src/components/lyrics/edit-modal.tsx

```
@@ -0,0 +1,90 @@
+    onOpenChange(false);
+  }, [onOpenChange]);
+
+  return (
+    <Modal
+      isOpen={isOpen}
+      onOpenChange={onOpenChange}
+      scrollBehavior="inside"
+      size="2xl"
+    >
+      <ModalContent>
+        {(onClose) => (
```

ESLint

[error] Replace `(onClose)` with `onClose`

(prettier/prettier)